libcudacxx/include/cuda/std/detail/libcxx/include/__config

```diff
@@ -545,7 +545,7 @@ typedef __char32_t char32_t;
 // Try to find out if RTTI is disabled.
 // g++ and cl.exe have RTTI on by default and define a macro when it is.
 #  ifndef _LIBCUDACXX_NO_RTTI
-#    define _LIBCUDACXX_NO_RTTI
+#    define _LIBCUDACXX_NO_RTTI _CCCL_NO_RTTI
 #  endif // !_LIBCUDACXX_NO_RTTI
```

Comment:
Again, I am totally in favor of dropping libcu++ specific things in favor of global cccl macros

Reply:
i avoided making that change in this PR because it would make it harder to review. but i can make the change if you're not bothered by that.